backend/module/sourcecontrol/sourcecontrol.go

```diff
@@ -31,22 +42,104 @@ func New(*any.Any, *zap.Logger, tally.Scope) (module.Module, error) {
 		return nil, errors.New("service was not the correct type")
 	}
 
-	return &mod{c}, nil
+	return &mod{c, config}, nil
 }
 
 type mod struct {
 	github github.Client
+	config *sourcecontrolconfigv1.Config
 }
 
 func (m *mod) Register(r module.Registrar) error {
 	sourcecontrolv1.RegisterSourceControlAPIServer(r.GRPCServer(), m)
 	return r.RegisterJSONGateway(sourcecontrolv1.RegisterSourceControlAPIHandler)
 }
 
-func (m *mod) CreateRepository(ctx context.Context, req *sourcecontrolv1.CreateRepositoryRequest) (*sourcecontrolv1.CreateRepositoryResponse, error) {
-	return m.github.CreateRepository(ctx, req)
+func contains(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
+
+func (m *mod) ManageableOrganizations(ctx context.Context) ([]*sourcecontrolv1.Entity, error) {
+	organizations, err := m.github.ListOrganizations(ctx, "")
+	writableOrgs := []*sourcecontrolv1.Entity{}
+	if err != nil {
+		return nil, err
+	}
```

Comment:
nit: move error under the `m.github.ListOrganizations` call

Reply:
I was trying to avoid having the logic of writable orgs within the list orgs call because you could technically use the github service to see all of a users orgs just for membership purposes. I could create a new function that is just `ListManageableOrganizations` or could have a parameter to control which orgs to return based on permissions. what do you think?